init/docker/docker-compose.env.example

```diff
@@ -11,6 +11,8 @@ GRAFANA_PASSWORD=grafanaadmin
 #unifi-poller
 POLLER_TAG=latest
 POLLER_DEBUG=false
+POLLER_SAVE_IDS=false
```

Comment:
Please don't include IDS. It's a failed attempt and the data is useless. If someone really wants it they can jump through an extra hoop. DPI is good though, thanks for the contribution!